portfolio/src/main/webapp/script.js

```diff
@@ -140,11 +145,17 @@ function createCommentElement(comment, id) {
   emailElement.addClass("comment-email");
   emailElement.attr("href", "mailto:" + comment.email).text(comment.email);
 
-  const deleteBtn = $("<button></button>");
-  deleteBtn.text("Delete");
-  deleteBtn.on("click", async () => {
-    await deleteComment(id);
-  })
+  // delete button only present if person logged in is teh one who posted the comment
+  let deleteBtn; 
```

Comment:
Instead of setting deleteBtn to be button/null, you could also just `node.append(deleteBtn)` within the if-statement.